comum/setSaveIn.pm

```diff
@@ -2,6 +2,7 @@ macro SetSaveIn {
     call pararDeAtacar
     do conf lockMap none
     do conf -f saveMap_storage_sequence c r1
+	do conf -f saveMap_save_sequence c r0
```

Comment:
troque o tab por 4 espaços

Reply:
ah então era isso que eu tava esquecendo... sempre dava uma linha vermlha quando salvava e eu nunca fui procurar...

o ideal num futuro próximo é mudar a sequencia de conversação do autostorage dependendo se for thor ou valhalla, ja que muda de um servidor para outro